client/src/components/StudentProfile/styles.css

```diff
@@ -1,7 +1,8 @@
-.studen-profile-container {
+.student-profile-container {
     min-height: calc(100vh - 100px);
     display: flex;
     margin: 50px auto;
+    width:  960px;
```

Comment:
This one should be max-width, but easily fixable later when we go through all the CSS